webpack.config.base.js

```diff
@@ -43,6 +46,9 @@ module.exports = {
     new ExtractTextPlugin({
       filename: '[name].css',
       allChunks: true
+    }),
+    new webpack.EnvironmentPlugin({
+      BACKEND_URL: process.env.BACKEND_URL,
```

Comment:
@Spetastian How does this work in production?  Want to add something to `README.md` about that?